src/zeusops_bot/cli.py

```diff
@@ -60,9 +64,12 @@ def main():
         print("Error while loading the bot's config from envvars", file=sys.stderr)
         raise
 
-    logging.basicConfig(level=logging.DEBUG)
-    logging.getLogger("discord").setLevel(logging.INFO)
-    logging.getLogger("discord.gateway").setLevel(logging.WARNING)
+    if args.debug:
+        logging.basicConfig(level=logging.DEBUG)
+    else:
+        logging.basicConfig(level=logging.INFO)
+        logging.getLogger("discord").setLevel(logging.INFO)
+        logging.getLogger("discord.gateway").setLevel(logging.WARNING)
 
```

Comment:
Should we make it a func to deal with it?

Reply:
Split the log setup into a function in a separate module. Was debating if the function should return the logger that's later passed to `ZeusopsBot`, but figured that that's not the job of the setup function.